mesheryctl/internal/cli/root/system/error.go

```diff
@@ -146,3 +147,13 @@ func ErrFailedGetEphemeralPort(err error) error {
 		nil, nil,
 	)
 }
+
+func ErrCreatingDockerClient(err error) error {
+	return errors.New(
+		ErrCreatingDockerClientCode,
+		errors.Critical,
+		[]string{"Failed to create Docker client"},
+		[]string{"Error occurred while creating Docker client from config file", err.Error()},
+		[]string{"Missing or invalid docker config"},
+		[]string{"Please check the Docker config file for any errors or missing information. Make sure it is correctly formatted and contains all the required fields."})
+}
```

Comment:
@Azanul there's nothing to mention about docker contexts, huh?

Reply:
Thanks. What if we attempt to use an invalid context? What if they have no valid context? Users don't need to confirm or verify their contexts, huh?

Reply:
mesheryctl doesn't inform the user of the attempted use of a context and the state of that failure?

Reply:
In case of invalid context his `docker ps` won't work either